engine/access/rest/routes/router.go

```diff
@@ -101,6 +101,11 @@ var Routes = []route{{
 	Pattern: "/accounts/{address}",
 	Name:    "getAccount",
 	Handler: GetAccount,
+}, {
+	Method:  http.MethodGet,
+	Pattern: "/accounts/{address}/keys/{keyID}",
+	Name:    "getAccountKeyByID",
```

Comment:
Updated in https://github.com/onflow/flow-go/pull/4583/commits/29402e349eb8e6b86a5da8f4e530c2a817e09dea
**Note:** I did not touch the regex, since I was able to contain the changes in the `normalizeURL` function and I added relevant test cases. Let me know if that is ok.